astropy/cosmology/io/tests/test_json.py

```diff
@@ -127,9 +130,13 @@ def test_readwrite_json_subclass_partial_info(self, cosmo_cls, cosmo, read,
         # partial information
         with open(fp) as file:
             L = file.readlines()[0]
-        L = L[: L.index('"cosmology":')] + L[L.index(", ") + 2 :]  # remove cosmology  # noqa: #203
+        L = (
```

Comment:
A little gross, but fine for now.

Reply:
Again we can `# fmt: skip` if you would like.

Reply:
No, I'd rather rewrite it to be more explicit about the string magic. An endeavor for a future pull request.